File: Aula_1/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void){
    float money;
    float coins = 0;

    do{
        money = get_float("Change owned: \n");
    }
    while(money < 0);

    float cents = round(money * 100);

    while (cents >= 25){
        cents = cents - 25;
        coins++;
    }

    while (cents >= 10){
            cents = cents - 10;
            coins++;
    }

    while (cents >= 5){
                cents = cents - 5;
                coins++;
    }

    while (cents > 0){
                    cents = cents - 1;
                    coins++;
    }


    printf("%.0f\n", coins);


}

File: Aula_6/Cash.py
from cs50 import get_float

def main():
    cash = change()*100
    coins = 0

    while cash >= 25:
        cash = cash - 25
        coins += 1
    while cash >= 10:
        cash = cash - 10
        coins += 1
    while cash >= 5:
        cash = cash - 5
        coins += 1
    while cash >= 1:
        cash = cash - 1
        coins += 1

    print(coins)



def change():
    while True:
        money = get_float("Change owned: ")
        if money > 0:
            break
    return money

main()